### DISCUSSION TITLE
Lists order Popular - how to get it via API

### DISCUSSION THREAD

**Discussion by linaspurinis** [trakt/trakt-api#328] [Q&A] [answered]
Hi,
On trakt.tv site, lists are ordered by "Popular". For example:
https://trakt.tv/movies/house-of-gucci-2021/lists

Is it possible to get this "Popular" rating via API? ~~Or is it calculated using likes and comments?~~

Thank you.

**Comment by rectifyer** [comment, marked as answer]
Yes, you can specify `popular` for the sort parameter.

https://trakt.docs.apiary.io/#reference/movies/lists/get-lists-containing-this-movie
